src/components/breeding.html

```diff
@@ -117,7 +117,7 @@ <h5 class="modal-title" id="breedingModalLabel">Day Care</h5>
                         <ul class="row justify-content-center p-0" data-bind="foreach: App.game.party.caughtPokemon">
                             <li class="eggSlot col-sm-4 col-md-3 col-lg-2 pokedexEntry text-nowrap" data-bind="visible: BreedingController.visible($data), style:{background: PokedexHelper.getBackgroundColors($data.name)}, class: App.game.breeding.hasFreeEggSlot() || App.game.breeding.hasFreeQueueSlot() ? '' : 'disabled'">
                                 <span style="top: 0; border-top-left-radius: 6px; border-top-right-radius: 6px;" data-bind="text: $data.name">name</span>
-                                <div data-bind="visible: App.game.party.alreadyCaughtPokemon($data.id, true)" style="position: absolute;right: 0px;top: 0px;">✨</div>
+                                <div data-bind="visible: App.game.party.alreadyCaughtPokemon($data.id, true)" style="position: absolute;right: 0px;top: 0px;">🌟</div>
```

Comment:
I'd rather keep the sparkles instead of the star for these please.

Reply:
I'll either revert this or see what I can do to the Sparkles emoji to keep it consistent but unique somehow. Leave it with me.